contributor-docs/adrs/adr-008-experimental-components.md

```diff
@@ -47,15 +47,15 @@ The approach that has served us well since Dec 2021 has been "drafts" (along wit
 
 1. npm package: "Experimental" components, as defined by the component lifecycle, should not be part of semantically versioned npm package for `@primer/react`.
 
-2. npm package: Each experimental component should be independently published and versioned with semver. Breaking changes should be tagged as major versions. Example: `@github/react-commentbox`
+2. npm package: Each experimental component should be independently published and versioned with semver. Breaking changes should be tagged as major versions. Example: `@github/experimental-react-commentbox`
 
-3. The code for experimental components should live in [github/github/modules/react-shared](https://github.com/github/github/tree/master/app/assets/modules/react-shared) and published as individual packages as suggested in point 2.
+3. The code for experimental components should live in a new repository, example: [github/react-experimental](https://github.com/github/react-experimental) and published as individual packages as suggested in point 2.
 
-   The monolith already has a place where reusable react components live. This has the lowest barrier to entry for folks working in the monolith.
```

Comment:
This is the relevant diff, [split view recommended](https://github.com/primer/react/pull/2398/files?diff=split&w=0#diff-5e30a807664dafbee9f227901a2459809a2c2226e183880f8bf6e05b1f52a7ceL52-L54)